packages/contact-center/store/src/store.ts

```diff
@@ -1,8 +1,16 @@
 import {makeAutoObservable, observable} from 'mobx';
 import Webex from 'webex';
-import {IContactCenter, IAgentProfile, Team} from '@webex/plugin-cc';
+import {
+  AgentLogin,
+  IContactCenter,
+  IAgentProfile,
```

Comment:
We might need to update this as with the latest sdk changes in Ravi's PR, we use `Profile` instead of `IAgentProfile`.